### PR TITLE
Result

### PR DESCRIPTION
**Pull request by prafullatass**
Fixes # 36

# Fetching Instructions
```shell
git fetch --all
git checkout result
```

# Description
Added new views to display the final score.  
When the user click on the 'view leaderboard' button, the initial page will load which has an input to enter initials
When the user click on the 'Exit' link, the thank you page will load

For testing use the link: http://localhost:3000/score

# List of changes
- 
- 

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation specific to my parts
- [ ] My changes generate no new warnings
- [ ] All tests pass with no new warnings

### CODE REVIEW

File: browser/src/components/views/Initials.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <div className="initialsContainer">
+          <Button className="closeBtn" color="link" onClick={() => this.props.history.push('/thanks')}>Exit</Button>
+          <h1 className="mt-5 score"> {score} pts </h1>
+          <p className="mt-2 scoreText"> YOUR SCORE </p>
+          <hr />
+          <h3 className="mt-2 text-center"> Add your initials to the leaderboard </h3>
+          <Form>
+            <FormGroup>
+              <Label className="m-3" for="initials">Add three letter</Label>
+              <Input type="text" name="initials" id="initials" placeholder="initials" maxLength="3" autocomplete="off" />
```
Comment (drteresavasquez) [inline browser/src/components/views/Initials.js:19]: this needs to be `autoComplete`

**Review by drteresavasquez — CHANGES_REQUESTED** [review]
other than one small change, this looks great!

**Comment by ankajgoel** [comment]
Screen Shot 2020-10-29 at 9 59 35 PM
on http://localhost:3000/initials the button name is view leaderboard, based on Invision this would be 'Add my score'

File: browser/src/components/views/Initials.js (hunk 1 above)
Comment (prafullatass) [reply browser/src/components/views/Initials.js:19]: done..

**Comment by drteresavasquez** [comment]
Merged into main branch, so updates are on development

**Pull request closed without merging**